content/en/service_management/events/guides/email.md

```diff
@@ -124,3 +133,6 @@ The maximum allowed email size, including content and attachments, is 20MB. Emai
 [3]: https://app.datadoghq.com
 [5]: http://daringfireball.net/projects/markdown/syntax#lin
 [6]: /integrations/guide/events-from-sns-emails/
+
+### Usage tracking
+In order to help understand which emails are being used and receiving events, there is a column in the **Events API emails** called `Last Used`. That date reflects the last time an email was processed for each address, or `No data` when there are no records of it being used.
```

Comment:
```suggestion
To understand which emails are being used and receiving events, check the `Last Used` column in the **Events API Emails** tab in organization settings. This displays the most recent date that an email was processed for each address, or `No data` if there are no records of it being used.
```

Reply:
In addition to the suggestions here, this section should be moved above the list of reference links from lines `131` - `135`